docs/MigrationTimeline.md

```diff
@@ -0,0 +1,101 @@
+# Migration Timelines
+
+## 15 day
+
+This 15-day migration timeline demonstrates an example of a migration process that can be adapted to 3 working weeks when including weekends. Key phases:
+
+1. Deployment and setup (Days 1-5)
+1. Historical backfill and catchup (Days 5-10)
+1. Validation and teardown (Days 11-15)
+
+### Timeline
+
+```mermaid
+%%{
+  init: {
+    "gantt": {
+        "fontSize": 20,
+        "barHeight": 40,
+        "sectionFontSize": 24,
+        "leftPadding": 175
+    }
+  }
+}%%
+gantt
+    dateFormat D HH
+    axisFormat Day %d
+    todayMarker off
+    tickInterval 1day
+
+    section Steps
+    Deployment : milestone, deployment, 1 00, 0d
+    Setup and Verification : setup, after deployment, 5d
+    Clear Test Environment : milestone, clear, after setup, 0d
+    Traffic Capture : traffic_capture, after clear, 5d
+    Snapshot : snapshot, after clear, 1d
+    Metadata Migration : metadata, after snapshot, 1h
+    Scale Up Target Cluster : milestone, after metadata, 0d
+    Reindex from Snapshot : rfs, after metadata, 71h
+    Scale Down Target Cluster for Replay : milestone, after rfs, 0d
+    Traffic Replay: replay, after rfs, 1d
+    Traffic Switchover : milestone, switchover, after replay, 0d
+    Validation : validation, after switchover, 3d
+    Scale Down Target Cluster : milestone, 12 00, 0d
+    Teardown   : teardown, after validation, 2d
+```
+
+#### Explanation of Scaling Operations
+
```

Comment:
This section was useful, but it feels like it could use a few sentences up front explaining that we assume the user will deliberately overscale the target to speed up the migration, why they might do that, and then lead into the existing discussion.  I get that you mention that briefly in the first bullet point but I think it would be better to extend that into a bit larger of a context.

Reply:
Updated, thanks